src/System.Management.Automation/engine/GetEvent_Types_Ps1Xml.cs

```diff
@@ -1,90 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@daxian-dbw, your last commit had 1 failures in `PowerShell-CI-windows`
Task-based PowerShell async APIs.PowerShell::StopAsync.can stop a script that is running asynchronously
```powershell
Expected $true, because Result                  : 
Id                      : 1
Exception               : System.AggregateException: One or more errors occurred. (The pipeline has been stopped.)
                           ---> System.Management.Automation.PipelineStoppedException: The pipeline has been stopped.
                             at System.Management.Automation.Runspaces.AsyncResult.EndInvoke()
                             at System.Management.Automation.PowerShell.EndInvoke(IAsyncResult asyncResult)
                             at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
                             --- End of inner exception stack trace ---
Status                  : Faulted
IsCanceled              : False
IsCompleted             : True
IsCompletedSuccessfully : False
CreationOptions         : None
AsyncState              : 
IsFaulted               : True
AsyncWaitHandle         : System.Threading.ManualResetEvent
CompletedSynchronously  : False, but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Api\TaskBasedAsyncPowerShellAPI.Tests.ps1: line 230
230:                 $ir.IsFaulted | Should -Be $true -Because ($ir | Format-List -Force * | Out-String)
```